src/main/java/net/sourceforge/tess4j/util/PdfUtilities.java

```diff
@@ -71,46 +75,35 @@ public static File convertPdf2Tiff(File inputPdfFile) throws IOException {
      * @param inputPdfFile input file
      * @return an array of PNG images
      */
-    public static File[] convertPdf2Png(File inputPdfFile) {
+    public static File[] convertPdf2Png(File inputPdfFile)  {
         File imageDir = inputPdfFile.getParentFile();
```

Comment:
In case parent directory is being used as the working directory for multiple PDF files in the same directory, it may cause the extracted data to get garbled, or other file level exceptions when accessed by multiple threads at the same same time. Possible solution is to create individual working directories per PDF file in the parent directory, having same name as the PDF file, thereby maintaining uniqueness. The PNG images of that PDF file would be created in this new working directory, and thereby avoid conflicts. 
Sample Code : 
```
File imageDir = new File(inputPdfFile.getParentFile()+System.getProperty("file.separator")+inputPdfFile.getName().replace(".pdf", ""));
    	imageDir.mkdirs();
```